### DISCUSSION TITLE
Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Apr 6, 2023

### DISCUSSION THREAD

**Discussion by momo-clown** [community/community#52145] [GitHub Education]
I am a student at Yunnan University, which is located in Kunming City, Yunnan Province, China. I have submitted my application more than 20 times, but each time it has been rejected. I have provided my student ID card, transcripts, student card used in school, academic certificates certified by the Ministry of Education of China, and even my acceptance letter. However, I still receive a response saying that my student identity cannot be confirmed. I have submitted all the materials that can prove my student identity, but still have not been able to pass the application process. Furthermore, I am unable to upload clear images because I have to take pictures using my browser's camera, which produces low-quality images. Even after taking photos with my phone and uploading them, my application is still rejected. I hope to seek help in this matter.

**Comment by samus-aran** [comment]
Hi @momo-clown 

What was your last rejection reason and we can start from there.

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.
